package.json

```diff
@@ -95,85 +93,77 @@
     "compile": "tsc",
-    "compile:tests": "tsc --project tsconfig.test.json",
-    "jasmine:start": "cross-env NODE_ICU_DATA=node_modules/full-icu jasmine --config=jasmine.json",
-    "check:licenses": "license-checker --production --excludePackages=\"hyperformula@0.0.1\" --onlyAllow=\"MIT; Apache-2.0; BSD-3-Clause; BSD-2-Clause; ISC; BSD; Unlicense\"",
+    "check:licenses": "license-checker --production --excludePackages=\"hyperformula@2.7.1\" --onlyAllow=\"MIT; Apache-2.0; BSD-3-Clause; BSD-2-Clause; ISC; BSD; Unlicense\"",
     "tsnode": "ts-node --transpile-only -O {\\\"module\\\":\\\"commonjs\\\"}"
   },
   "dependencies": {
     "chevrotain": "^6.5.0",
     "tiny-emitter": "^2.1.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/register": "^7.9.0",
-    "@babel/runtime": "^7.18.9",
-    "@babel/preset-react": "^7.13.13",
-    "@babel/preset-typescript": "^7.13.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
+    "@babel/cli": "^7.25.9",
+    "@babel/core": "^7.26.0",
+    "@babel/plugin-transform-runtime": "^7.25.9",
+    "@babel/preset-env": "^7.26.0",
+    "@babel/register": "^7.25.9",
+    "@babel/runtime": "^7.26.0",
+    "@babel/preset-react": "^7.25.9",
+    "@babel/preset-typescript": "^7.26.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.25.9",
     "@babel/plugin-syntax-class-properties": "^7.12.13",
-    "@babel/plugin-proposal-decorators": "^7.13.5",
-    "@babel/plugin-proposal-class-properties": "^7.13.0",
-    "@babel/plugin-proposal-private-methods": "^7.13.0",
-    "@babel/plugin-proposal-private-property-in-object": "^7.13.0",
-    "@babel/plugin-transform-typescript": "^7.24.7",
-    "@microsoft/tsdoc": "^0.12.16",
-    "@types/jasmine": "^4.0.0",
-    "@types/jest": "^26.0.0",
-    "@types/jsdom": "^16.2.1",
-    "@types/node": "^10.17.19",
-    "@types/webpack-env": "^1.15.2",
-    "@typescript-eslint/eslint-plugin": "^5.59.1",
-    "@typescript-eslint/parser": "^5.59.1",
-    "@vuepress/plugin-active-header-links": "^1.3.1",
+    "@babel/plugin-proposal-decorators": "^7.25.9",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@types/jasmine": "^5.1.4",
+    "@types/jest": "^26.0.24",
+    "@types/jsdom": "^21.1.7",
+    "@types/node": "^17.0.45",
+    "@types/webpack-env": "^1.18.5",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
     "as-table": "^1.0.55",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-transform-inline-environment-variables": "^0.4.3",
-    "core-js": "^3.23.0",
-    "cross-env": "^7.0.0",
+    "babel-loader": "^8.4.1",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.4",
+    "core-js": "^3.39.0",
```

Comment:
After bumping the `core-js,`, you should also update the version in [babel config](https://github.com/handsontable/hyperformula/blob/feature/issue-1451/babel.config.js#L12).

Reply:
Done. Thank you for spotting it ;)